subresource-loading.md

```diff
@@ -72,17 +72,25 @@ Content blockers have a number of requirements when it comes to ensuring that ba
 - Resource bundles must not enable the cheap rotation of URLs within the bundle, as this would make URL-based content blocking much more difficult. (c.f. ["Personalization"](https://github.com/littledan/resource-bundles/blob/main/subresource-loading.md#personalization))
 - When content is blocked, it's ideal if sites don't tend to cause browsers to download the blocked content. (c.f. ["code splitting"](#for-web-developers))
 
-## One design: Two-level chunking with client-side hints manifest
+## Proposals
 
-Below is just one concrete place in the design space, as a concrete starting point for discussion and prototyping. There are many possible alternatives to continue considering, discussed in the Design FAQ below.
+Below is a set of potential concrete places in the design space, as concrete starting points for discussion and prototyping. There are many possible alternatives to continue considering, discussed in the Design FAQ below.
+
+### Proposal #1: Two-level chunking with client-side hints manifest
+
+This proposal assumes the following:
+* The site's code is bundled into a single "mega bundle".
+* The bundle declaration on each entry points contains a *complete* list of routes and their dependencies.
+  - Specifically, the browser is not aware of the full list of resources the bundle contains, just of the top-level routes. If that list is not complete, secondary resources can be discovered by the browser before the bundle's index is fetched, resulting in spurious downloads.
```

Comment:
It would result in 404s, not spurious downloads.

Reply:
If a markup-based dependency for the current page is missing from the list, the browser doesn't know that it's in the bundle, and may request it before knowing that, no?

Reply:
Perhaps just the "explicitly declared chunks"?

Reply:
> If a markup-based dependency for the current page is missing from the list, the browser doesn't know that it's in the bundle, and may request it before knowing that, no?

Anything in the scope would be treated as "comes from the bundle" (unless the browser decides to opt out of that and do individual requests). So, if there are no pending bundle fetches, then the browser can issue a 404, to make things easier to debug (the same reasoning as the same decision was taken in step 4 of https://github.com/WICG/webpackage/blob/master/explainers/subresource-loading.md#link-based-api ).

Reply:
OK, in this case, I'll remove this point

Reply:
Replaced it with "resources in the bundle's scope must come from the bundle"